resource-definitions/terraform-driver/dynamic-credentials-from-config/aws-file-credentials.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: entity.humanitec.io/v1b1
+kind: Definition
+metadata:
+  id: aws-file-credentials
+entity:
+  # Reference an account from a config resource to allow
+  # using specific credentials for different environments
+  driver_account: ${resources.config#aws-account.account}
+  driver_inputs:
+    # Use the credentials injected via the driver_account
+    # to set variables as expected by your Terraform code
+    credentials_config:
+      file: aws_creds
```

Comment:
@delca85 Could you please confirm that I'm using `file` option correctly here (and in GCP example too)